ui/app/src/views/CreatePoolPage.vue

```diff
@@ -282,19 +278,41 @@ export default defineComponent({
       <template #header>Pool Token Prices</template>
       <template #body>
         <FatInfoTableCell>
-          <span class="number">{{ formatNumber(aPerBRatioMessage === 'N/A' ? '0' : aPerBRatioMessage) }}</span
+          <span class="number">{{
+            formatNumber(aPerBRatioMessage === "N/A" ? "0" : aPerBRatioMessage)
+          }}</span
           ><br />
           <span
-            >{{ fromSymbol.toLowerCase().includes("rowan") ? fromSymbol.toUpperCase() : "c" + fromSymbol.slice(1).toUpperCase() }} per
-            {{ toSymbol.toLowerCase().includes("rowan") ? toSymbol.toUpperCase() : "c" + toSymbol.slice(1).toUpperCase() }}</span
+            >{{
+              fromSymbol.toLowerCase().includes("rowan")
+                ? fromSymbol.toUpperCase()
+                : "c" + fromSymbol.slice(1).toUpperCase()
+            }}
+            per
+            {{
+              toSymbol.toLowerCase().includes("rowan")
+                ? toSymbol.toUpperCase()
+                : "c" + toSymbol.slice(1).toUpperCase()
```

Comment:
Will get this cleaned up asap next week. 

reminder to change all the `from` `to` semantics.

Reply:
cool!